_docs/_latest/setup/install/roles-and-permissions.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Service Account roles and permissions
```

Comment:
Rename to Service Account roles and required APIs? This page only talks about roles (no permissions), and also includes the APIs.

Reply:
Sounds good, renamed.